extensions/amp-analytics/0.1/vendors/kenshoo.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const KENSHOO_CONFIG = /** @type {!JsonObject} */ ({
+  'vars': {
+    'clientId': 'CLIENT_ID(ken_amp_id)',
+    'channelClickId': 'QUERY_PARAM(gclid)',
+    'tid': 'KT-XXXXX-XXX',
+    'userId': '<USER_ID>',
```

Comment:
Do you want this to be the default value if a publisher misconfigures the document?

Reply:
@calebcordry Yes
@ifatgv4 FYI